variants/heltec_wireless_paper/pins_arduino.h

```diff
@@ -65,6 +61,6 @@ static const uint8_t LED = 18;
 
 static const uint8_t RST_LoRa = 12;
 static const uint8_t BUSY_LoRa = 13;
-static const uint8_t DIO0 = 14;
+static const uint8_t DIO1 = 14;
```

Comment:
This constant actually isn't used by Meshtastic code-base. Board schematic does show DIO1 connected to GPIO14 though, not DIO0.